src/Config.php

```diff
@@ -1,51 +1,52 @@
 <?php
+
 namespace Pantheon\Terminus;
 
 use Dotenv\Dotenv;
 use Symfony\Component\Yaml\Yaml;
 
 class Config
 {
-  /**
-   * @var string[]
-   */
+    /**
```

Comment:
PSR-2 is four spaces, so I would vote "yes".  Don't remember if you adopted PSR-2 yet, though.